### ISSUE TITLE
ORP and PH readings

### ISSUE THREAD

**Issue by art0** [DutchForce75/pyZodiacTri#1]
Great stuff!
Did you know if it is possible to get the ORP and PH readings? 
Holding the SELECT key in the keypad there is a hidden menu with a few settings. One of them allows to read the PH and ORP. 

The Zodiac only works when CYA is under 50 ppm in the pool and values between 500 mV and 750mV. Once the ORP is smaller than 500 mV the SWG is pretty useless.

**Comment by DutchForce75** [comment]
Yes, I have found that out, but I'd really would like to read those results byt the RS485 line as well. No luck so far!

Tried to do fire some Chemlink commands after connecting, but that didn't do the trick. Might be that the AquaLink protocol might have implemented that, but I can only try that in a couple of weeks from now.

**Comment by art0** [comment]
The Salt-level and the Add Salt values remain "-". Did you make it work?

I am considering building something using these sensors for ORP and PH https://www.atlas-scientific.com/

I am not sure there is a simple way to read those values. You know about the hidden menu, right?

**Comment by art0** [comment]
A bit more feedback:

- In the panel i can only see the "Blue LED" when it goes into BOOST mode, when setting the % of Chlore i do not see anything in the panel. I was expecting something like OUTPUT: X%

- No values from Salt, Add-Salt are displayed. Happy to send you the RS-debug of the command.

**Comment by DutchForce75** [comment]
Yes I know about the secret ph and orp menu, so it must be possible to get it out somehow.

My display does change it's Output setting when a command is sent, and also gives me back the salt levels. My guess is that you are havent selected Jandy Rev L/M as the controller?
